### ISSUE TITLE
Trying to use ssl in a request doesn't work.

### ISSUE THREAD

**Issue by linus-jansson** [whoshuu/cpr#423]
I can't seem to get https to work. Trying to use the same url with http works but as soon as I change the Url to https the program runs and gives a status_code 0 and returns in less than 0.02 seconds with no body. No error codes or anything. I would assume that the library doesn't send my request to the url at all because of the time it took for it to return, the status code and no body.

Can it be that im missing something. Anyone got an idea?

Removing the ssl option does not change anything.
```c++
auto ssl = cpr::Ssl(cpr::ssl::TLSv1_2{});
    cpr::Response response = cpr::Get(cpr::Url{"https://www.httpbin.org/get"},
                                      cpr::Parameters{{"hello", "world"}},
                                      ssl);

    if (response.status_code >= 400)
    {
        std::cerr << "Error [" << response.status_code << "] making request" << std::endl;
    }
    else
    {
        std::cout << "Request took " << response.elapsed << std::endl;
        std::cout << "Body:" << std::endl
                  << response.text;
    }
```

output;
```
Request took 0.013933
Body:
```

**Comment by linus-jansson** [comment]
I have now printed the response error message from the connection error. It returns with a "SSL certificate problem: unable to get local issuer certificate". Which lead me to believe that I need to specify the SSL certificates. But I am unsure which ones and how to do so efficiently if you would want to release it as a library building on cpr. 
Shouldn't this be something that curl does in the background?

**Comment by linus-jansson** [comment]
A little short looked by me. But I found the other closed issues about this  for example in #261 for now I will just go with disabling the verifySsl until I find some better solution.
